internals/core_models.py

```diff
@@ -486,3 +488,9 @@ class Stage(ndb.Model):
   announcement_url = ndb.StringProperty()
   # Origin trial stage id that this stage extends, if trial extension stage.
   ot_stage_id = ndb.IntegerProperty()
+
+  #Enterprise
+  rollout_milestone = ndb.IntegerProperty()
+  rollout_platforms = ndb.StringProperty(repeated=True)
+  rollout_details = ndb.StringProperty()
```

Comment:
If this can be a paragraph of text, it would be better to make it a ndb.TextProperty().  The problem with StringProperty is that it is limited to only 1500 bytes.